addon/services/scroll.js

```diff
@@ -2,6 +2,7 @@ import Service, { inject as service } from '@ember/service';
 import { action } from '@ember/object';
 import { scheduleOnce, next } from '@ember/runloop';
 import { guidFor } from '@ember/object/internals';
+import config from 'ember-get-config';
```

Comment:
https://github.com/briarsweetbriar/ember-get-config/blob/master/index.js#L27 – really?

Reply:
We could get it through the container in this case if that's preferable (as we access it in a service). I don't see much downside to using ember-get-config either. Many addons utilise it and chances an app already have the dependency is huge.

Reply:
I think I'm generally not a fan of micro-libraries – this seems a bit like left-pad to me :)